package/CHANGELOG

```diff
@@ -14,11 +14,17 @@ The rules for this file:
 
 
 -------------------------------------------------------------------------------
-??/??/?? IAlibay
+11/30/24 IAlibay, tanishy7777
```

Comment:
```suggestion
??/??/?? IAlibay, tanishy7777
```
This is supposed to be the date of the next release, not your contribution. Please leave it unset.

Reply:
oh ok